api/candlepin-api-spec.yaml

```diff
@@ -428,6 +428,23 @@ components:
             value:
               type: string
 
+    CdnDTO:
+      description: DTO representing a Cdn
+      allOf:
+        - $ref: '#/components/schemas/TimestampedEntity'
+        - type: object
+          properties:
+            id:
+              type: string
```

Comment:
Because the current CdnDTO has @JsonProperty/@JsonIgnore set to getId/setId respectively, we should also add the `readOnly: true` property to this field. Currently it doesn't seem like it does anything at all, but I'm investigating if anything can be done.

Reply:
Alright, since we're dropping the use of @JsonIgnore entirely, lets just leave this as is